ci-operator/step-registry/mtr/interop-scenario/mtr-interop-scenario-chain.yaml

```diff
@@ -0,0 +1,24 @@
+chain:
+  as: mtr-interop-scenario
+  steps:
+  - ref: interop-tooling-operator-install
+  - ref: mtr-orchestrate
+  - ref: mtr-execute
+  - ref: mtr-report
+  env:
```

Comment:
I don't think this is needed. You are setting these env vars in the config. That should be enough to make use of them in the operator-install script.

Reply:
Yeah, I have a cleanup task set up. I am just trying to get past a few road blocks before I do some cleanup.

1. I will add them to the OWNERS files
2. I can give it a shot. Considering this is working for now, I don't want to make too many changes until I have a successful run
3. I will submit a PR to the windup_integration_test repo when I have successful runs
4. A lot of this needs to be in here. I will be changing it slightly to make it more sane